README.md

```diff
@@ -3,3 +3,9 @@
 Complete the tutorial at the link [here](https://sqlbolt.com/lesson/select_queries_introduction)
 
 Fork and clone this repo, then for each of the 18 sections, copy and paste your solution in your fork.  Submit by making a PR against this repo.
+
+1. SELECT title FROM movies
+2. SELECT director FROM movies
+3. SELECT title, director FROM movies
+4. SELECT title, year FROM movies
+5. SELECT title, director, year, Length_minutes FROM movies 
```

Comment:
You can have `*` to select all columns from a table.
Like...
`SELECT * 
  FROM movies`